lomap/gufe_bindings/network_generation.py

```diff
@@ -44,13 +44,13 @@ def generate_lomap_network(
        to 0.0 (worst).  These values are use as the "distance" between two molecules, and compared against the
        'distance_cutoff' parameter
     distance_cutoff : float
-       the maximum distance/dissimilarity between two molecules for an edge to be accepted
+       edges with a score >= 1 - distance_cutoff will be accepted. Default is 0.4.
```

Comment:
```suggestion
       edges with a score < 1 - distance_cutoff will be rejected. Default is 0.4.
```
Same thing, but it's closer to the idea of a `distance_cutoff` (It's a rather confusing name, we may rename).